implementations/rust/ockam/ockam_command/tests/bats/orchestrator/portals.bats

```diff
@@ -281,7 +280,7 @@ teardown() {
   socat_pid=$!
   sleep 2
 
-  run_success curl -sfI --retry-all-errors --retry-delay 2 --retry 10 -m 30 "127.0.0.1:${inlet_port}"
+  run_success curl -sfI --retry-all-errors --retry-delay 2 --retry 30 -m 60 "127.0.0.1:${inlet_port}"
```

Comment:
I was actually wondering all this time we even need retries. Now I'm even more concerned why we need to increase retry attempts and time. Isn't it supposed to work from the first try here?

Reply:
Yes, it's supposed to work on the first try, I forgot to revert inlet-related tests changes before pushing.
I'll address these tests in a follow-up PR if i manage to discover the root cause.